lib/nexpose/scan.rb

```diff
@@ -11,6 +11,16 @@ def scan_device(device)
       scan_devices([device])
     end
 
+    # Perform an ad hoc scan of a single device at a specific time.
+    #
+    # @param [Device] device Device to scan.
+    # @param [Array[adhoc_schedules]] list of scheduled times at which to run
+    # @return [Status] whether the request was successful
+    #
+    def scan_device_with_schedule(device, schedule)
```

Comment:
What's different between `scan_device_with_schedule` and `scan_asset_with_schedule`? Do we need both? Can we alias one or drop one altogether? I think we should prefer to use asset instead of device in general if we're providing asset IDs.